src/App.tsx

```diff
@@ -11,18 +10,15 @@ const Pokemon = lazy(
 );
 function App() {
   return (
-    <div
-      className="relative flex flex-col justify-center items-center w-full h-full object-fill  overflow-clip
-    "
-      object-fill
-    >
-      <div className="absolute z-0 overflow-auto object-cover ">
-        <Background />
-      </div>
-      <Header />
+    <div className="relative flex flex-col justify-center items-center w-screen h-screen object-fill ">
+      <Background />
+      <img src="../src/assets/logo.png" className=" z-20  h-20 w-48" />
       <BrowserRouter>
         <Suspense fallback={<Loader />}>
-          <div className="app flex flex-wrap z-10 justify-center align-middle">
+          <div
+            className="flex flex-wrap z-10 justify-center object-fill min-w-min overflow-scroll backdrop-opacity-60  h-5/6 w-5/6 bg-white align-middle   bg-white/30 no-scrollbar border-2 border-purple-600"
```

Comment:
Few things here as well

1. Flex-wrap should not be required at the root level. this should be handled per page level.
2. is zindex required here ? it should not be the case now.
3. Object-fill doesn't work on divs. remove that
4. Overflow should be handled at the page level. parent element should just occupy the required page height/width .. so remove that.
5. I don't think min-w-min is required.
6. Avoid using no-scrollbar unless its mentioned in  a products requirement .. because this hides any overflow behaviour issue that a page is having and is hard to identify during debugging as well. 
7. I don't think align-middle is required, if we have flex-box use its property to manipulate its children. 

```suggestion
            className="flex justify-center backdrop-opacity-60  h-5/6 w-5/6 bg-white   bg-white/30 nborder-2 border-purple-600"
```